src/app/features/tasks/recurring-interval/custom-interval/custom-interval.component.html

```diff
@@ -0,0 +1,16 @@
+
+<mat-dialog-content>
+  <span i18n>repeat every</span>
+  <input matInput type="number" [(ngModel)]="selectedValue" class="field-number" [min]="1">
```

Comment:
I dont like this component too much and entering something else then a number breaks the code

Reply:
I agree it's not terribly pretty yet ... Having a separate popup can be the basis for future extensions to allow more complex intervals, however.

Google Calendar handles it like this:
![image](https://user-images.githubusercontent.com/1682541/207128851-d8c5bb4e-fbb2-4dce-a3e9-cc91efaadbf7.png)

Reply:
I guess only Chrome prevents text from being entered. Added a validation/transformation now.